src/app/rundown-view/components/rundown-subheader/rundown-subheader.component.ts

```diff
@@ -0,0 +1,47 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges, SimpleChange, SimpleChanges } from '@angular/core'
+import { Rundown } from '../../../core/models/rundown'
+import { Piece } from '../../../core/models/piece'
+import { Tv2PieceMetadata } from '../../../core/models/tv2-piece'
+import { Tv2PieceType } from '../../../core/enums/tv2-piece-type'
+import { Part } from '../../../core/models/part'
+
+@Component({
+  selector: 'sofie-rundown-subheader',
+  templateUrl: './rundown-subheader.component.html',
+  styleUrls: ['./rundown-subheader.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class RundownSubheaderComponent implements OnChanges {
+  @Input()
+  public rundown: Rundown
+
+  public endWords?: string
+
+  public ngOnChanges(changes: SimpleChanges): void {
+    const rundownChange: SimpleChange = changes['rundown']
+    if (rundownChange) {
+      this.setEndWords()
+    }
+  }
+
+  private setEndWords(): void {
+    this.endWords = this.rundown.segments
+      .find(segment => segment.isOnAir)
+      ?.parts.find(part => part.isOnAir && this.doesPartContainVideoClipPiece(part))
+      ?.pieces.find(piece => this.isManusPiece(piece))?.name
+  }
```

Comment:
Do i understand this right, that you can't have a part with a manus piece that doesn't also contain a video clip piece?

Reply:
You can have a part with a manus piece but not a video clip piece – but it should not be shown as end words.

Reply:
Okay that explains the logic in place here. Thanks.